version: '3.7'
services:
  sync:
    depends_on:
      - auth
      - ari
      - confd
      - calld
      - rabbitmq
    environment:
      TARGETS: "ari:5039 rabbitmq:5672 confd:9486 auth:9497 calld:9500"

  ari:
    command: "python /usr/local/share/ari/mock_ari_fail.py 5039"
